certificate_agent.py

```diff
@@ -180,7 +187,8 @@ def main():
                 )
                 (download_uuid,
                  verify_uuid,
-                 download_url) = cert.create_and_upload(name.encode('utf-8'),
+                 download_url,
+                 download_url_png) = cert.create_and_upload(name.encode('utf-8'),
```

Comment:
@crdepa let's call it `png_url`.
In the same, more down at line 260:
`'png_url': png_url`